src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -205,9 +208,36 @@ stop_metadata_sync_to_node(PG_FUNCTION_ARGS)
 						errmsg("node (%s,%d) does not exist", nodeNameString, nodePort)));
 	}
 
+	if (NodeIsCoordinator(workerNode))
+	{
+		ereport(ERROR, (errmsg("node (%s,%d) is the coordinator and should have "
```

Comment:
This should be a NOTICE followed by return, otherwise the following type of commands would fail, which is not great:
```SQL
SELECT stop_metadata_sync_to_node(nodename,nodeport) from pg_dist_node;
ERROR:  node (localhost,5432) is the coordinator and should have metadata, skipping stopping the metadata sync
Time: 1.752 ms

```

Reply:
hmm, missed that. thank you